<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>appGalerie</class>
 <widget class="QMainWindow" name="appGalerie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/Logo_CIP_horizontal_res.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="vLayout_Center">
      <item>
       <layout class="QHBoxLayout" name="hLayout_Up">
        <item>
         <layout class="QVBoxLayout" name="vLayout_ULeft">
          <item>
           <widget class="QLabel" name="ImgPrincipal">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="vLayout_URight">
          <item>
           <widget class="QLabel" name="txtPhotographe">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="txtTitre">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hLayout_Center">
        <item>
         <widget class="QScrollArea" name="MiniatureScroll">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="MiniatureScrollWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>697</width>
             <height>443</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hLayout_Down"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
